node/resolvers/Mutations/CostCenters.ts

```diff
@@ -70,6 +71,7 @@ const CostCenters = {
         phoneNumber,
         sellers,
         stateRegistration,
+        paymentTerms
```

Comment:
Our lint is pointing an error on this line: `Error:   74:21  error    Insert ',' prettier/prettier`
Could you fix it before we merge, please?
```suggestion
        paymentTerms,
```